apertium/analysis/__init__.py

```
@@ -0,0 +1,30 @@
+from streamparser import parse
+
+import apertium
+from apertium.utils import to_alpha3_code, execute
+
+
+def postproc_text(result):
+    """
+    postprocesses the input
+    """
+    lexical_units = parse(result)
+    return list(lexical_units)[0]
+
+
+def analyze(text, lang):
+    """
+    runs apertium to analyze the input
+    """
+    in_text = text
+    in_mode = to_alpha3_code(lang)
+
+    if in_mode in apertium.analyzers:
+        path, mode = apertium.analyzers[in_mode]
+        formatting = 'txt'
+        commands = [['apertium', '-d', path, '-f', formatting, mode]]
+        result = execute(in_text, commands)
+        return postproc_text(result)
+
+    else:
+        return None
```

Nah, let's raise an Exception, here and below. Let's reserve None for something else.